src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/builders/MatchingAttribute.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.ida.verifyserviceprovider.builders;
```

Comment:
Seems weird to me for this to be in a 'builders' package.  I'll probably move it to a 'domain' package when I integrate this into the JSON schema story.